services/comm.h

```diff
@@ -85,6 +86,8 @@ enum MsgType {
     M_GET_CS,
     // S --> C
     M_USE_CS,  // = 'G'
+    // S --> CS
+    M_NO_CS,
```

Comment:
This reorders the enum and thus breaks compatibility.  You need to add to the end of this enum.